src/borderarray.jl

```diff
@@ -91,7 +91,7 @@ end
 
 @inline function _outeraxes(arr, border)
     map(axes(arr), border.lo, border.hi) do r, lo, hi
-        (first(r)-lo):(last(r)+hi)
+        Base.IdentityUnitRange((first(r)-lo):(last(r)+hi))
```

Comment:
```suggestion
        OffsetArrays.IdentityUnitRange((first(r)-lo):(last(r)+hi))
```

Reply:
Oh shoot, I thought I switched that, sorry.  Will fix now

Reply:
Should be fixed with the latest push (`OffsetArrays.IdentityUnitRange` was already imported so I didn't need any explicit prefix)